data/js/utility/content5.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+    //unhide "claim your prize!"
+    if (document.querySelector('.prize')) {
+        var prizeLink = document.querySelector('.prize');
+        prizeLink.classList.remove('hidden');
+
+        prizeLink.addEventListener('click', function() {
+        	addon.port.emit('intent', 'claimPrize');
+        });
+    }
+})();
```

Comment:
So, for the utility track, the user gets the badge on step5 just for opening the sidebar so, we can just have the prize link visible by default. We would therefore only need lines 4, and 7-9 here.